packages/ag-charts-enterprise/src/series/treemap/treemapSeriesModule.ts

```diff
@@ -22,48 +15,71 @@ export const TreemapSeriesModule: _ModuleSupport.SeriesModule<'treemap'> = {
     seriesDefaults: {},
     themeTemplate: {
         __extends__: EXTENDS_SERIES_DEFAULTS,
-        showInLegend: false,
+        fills: ['#c16068', '#a2bf8a', '#ebcc87', '#80a0c3', '#b58dae', '#85c0d1'],
         colorRange: ['#cb4b3f', '#6acb64'],
         group: {
             label: {
                 enabled: true,
-                color: DEFAULT_INVERTED_LABEL_COLOUR,
+                color: 'white',
```

Comment:
Nit: We should avoid hardcoded colours - should this change for dark-themes for example?